app/views/defra-id/register-your-interest/register-your-interest.njk

```diff
@@ -0,0 +1,65 @@
+{% extends './layouts/layout.njk' %}
+
+{% block pageTitle %}{{serviceName}}{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form method="POST" autocomplete="off" novalidate="novalidate">
+    <input type="hidden" name="crumb" value="{{crumb}}"/>
+
+    {% call govukFieldset({
+        legend: {
+          text: "Register your interest in a health and welfare review of your livestock",
+          classes: "govuk-fieldset__legend--l",
+          isPageHeading: true
+        }
+      }) %}
+
+    <br>
+    <p class="govuk-heading-body">To register your interest in applying for a review, enter the main business email address of the business registering. This is the main business email address linked to the business in your Rural Payments account.</p>
+    <p class="govuk-heading-body">Do not use your personal email address which may also be linked to the business.</p>
+
+    {{ 
+      govukInput({
+        label: {
+          text: "Main business email address",
+          classes: "govuk-label",
+          for: "emailAddress"
+        },
+        classes: "govuk-input govuk-!-width-three-quarters",
+        id: "emailAddress",
+        name: "emailAddress",
+        value: emailAddress,
+        errorMessage: errorMessages.emailAddress
+      }) 
+    }}
+
+    <p class="govuk-heading-body">The Rural Payments Agency will contact you with a link to start your application as soon as possible.</p>
+
+    <h2 class="govuk-heading-m">More than one business linked to customer reference number</h2>
+    <p class="govuk-heading-body">You’ll be able to choose which business you want a review for when you start your application.</p>
+
+    {% endcall %}
+
+    {{ 
+      govukButton({
+        classes: "govuk-!-margin-top-2 govuk-!-margin-bottom-8",
+        href: "/apply/register-your-interest/registration-complete",
+        isStartButton: true,
+        text: "Register",
+        preventDoubleClick: true
+      }) 
+    }}
+
+    <h2 class="govuk-heading-m">Help with registering</h2>
+
+    <p>If you need to update your main business email address, <a href="https://www.ruralpayments.service.gov.uk/">sign in to the Rural Payments service.</a></p>
+    <p class="govuk-heading-body">If you have another question about your registration, contact the RPA by email or phone:</p>
+
+    {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
+    
+    <p>If you do not have an account, <a href="https://www.ruralpayments.service.gov.uk/">register with the Rural Payments service.</a></p>
```

Comment:
rural payment service url should also be coming from the config and we can pass a value for it into this page